### PR TITLE
Use BACKGROUND_EXPORT_TASK_HANDLED_EXCEPTIONS to specify user handled exceptions

### PR DESCRIPTION
**Pull request by xkubov**

### CODE REVIEW

File: docs/installation.rst [hunk 1]
```diff
@@ -215,6 +215,15 @@ These configuration you can use with django-is-core in your django settings file
     def perform_action_before_export_task(request, queryset, filename, **kwargs):
         logger.info(f"Exporting result of the query ({queryset.query}) into the file '{filename}'.")
 
+.. attribute:: IS_CORE_BACKGROUND_EXPORT_TASK_HANDLED_EXCEPTIONS
```
Comment (rubickcz) [inline docs/installation.rst]:
```suggestion
.. attribute:: IS_CORE_BACKGROUND_EXPORT_TASK_CALLBACK_HANDLED_EXCEPTIONS
```
I would name it, as I suggested, `IS_CORE_BACKGROUND_EXPORT_TASK_CALLBACK_HANDLED_EXCEPTIONS` to indicate, that only exceptions, that arise from this callback call are relevant.

File: docs/installation.rst [hunk 2]
```diff
@@ -215,6 +215,15 @@ These configuration you can use with django-is-core in your django settings file
...
+
+  A list of exceptions declared as explicitly handled by the library user that may occur during
+  the background export. These exceptions do abort background export and trigger export_failed signal.
```
Comment (rubickcz) [inline docs/installation.rst]:
```suggestion
  the background export callback call. These exceptions do abort background export and trigger export_failed signal.
```

File: is_core/contrib/background_export/tasks.py [hunk 3]
```diff
@@ -101,12 +101,16 @@ def background_serialization(self, exported_file_pk, rest_context, language, req
...
-                exported_file=exported_file,
-            )
+            try:
+                import_string(settings.BACKGROUND_EXPORT_TASK_CALLBACK)(
+                    request=request,
+                    queryset=queryset,
+                    filename=filename,
+                    exported_file=exported_file,
+                )
+            except tuple(import_string(exc) for exc in settings.BACKGROUND_EXPORT_TASK_HANDLED_EXCEPTIONS) as exc:
```
Comment (rubickcz) [inline is_core/contrib/background_export/tasks.py]: I know that even the callback functionality is missing tests, but maybe it is time to add some. I'll leave this to your decision.

**Review by Formulka — APPROVED** [review]

**Review by rubickcz — CHANGES_REQUESTED** [review]

File: is_core/contrib/background_export/tasks.py (hunk 3 above)
Comment (xkubov) [reply is_core/contrib/background_export/tasks.py]: Hmm, I agree that tests would be useful here, however, adding them would be really cumbersome. The whole background export functionality is not tested, not just the callback. Apart from that, setting up tests locally is not particularly easy in this project.

I'll create a separate issue with a proper time estimate for the near future.

Comment (rubickcz) [reply is_core/contrib/background_export/tasks.py]: Yeah, I know, it would require some solid effort to get it working. Agreed.

**Review by rubickcz — APPROVED** [review]

**Review by rubickcz — APPROVED** [review]

**Pull request merged**